src/templates/forum/forum_form.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+
+{% load crispy_forms_tags %}
+{% load static i18n %}
+
+{% block title %}
+  {{ object.name }}
+{% endblock title %}
+{% block css %}
+  {{ block.super }}
+  <link href="{% static 'css/forums/forum_form.css' %}" rel="stylesheet" />
+{% endblock css %}
+{% block js %}
+  <script>
+    document.addEventListener("DOMContentLoaded", function(event) {
+      imageLabel = document.getElementById("image-label")
+      imageField = document.getElementById("id_image");
+      imageField.addEventListener('change', (e) => {
+        const file = e.target.files[0];
+
+        if (file) {
+          const trashButton = document.createElement('button');
+          const trashImg = document.createElement('i');
+          trashButton.classList.add("btn", "btn-danger");
+          trashButton.setAttribute("id", "trash-button");
+          trashImg.classList.add("fa-solid", "fa-trash-can")
+          trashButton.appendChild(trashImg);
+
+          // Change the text and styling when an image is uploaded...
+          imageLabel.classList.remove('btn-secondary');
+          imageLabel.classList.add('btn-warning');
+          imageLabel.innerText = "Change forum image";
+
+          // and add a trash can to remove the image
+          imageField.parentElement.appendChild(trashButton);
+          trashButton.addEventListener('click', removeSelectedImage);
+          trashButton.imageField = imageField;
+        }
+      });
+
+      // Removes the selected forum image
+      function removeSelectedImage(event) {
+        event.preventDefault();
+
+        // Clear the images
+        if (imageField && imageField.files[0]) {
+          imageField.value = "";
+          event.currentTarget.removeEventListener('click', removeSelectedImage);
+          event.currentTarget.remove();
+
+          // Remove the styles from the upload button
+          imageLabel.classList.remove('btn-warning');
+          imageLabel.classList.add('btn-secondary');
+          imageLabel.innerText = "Upload forum image";
```

Comment:
One extra comment here is that I cannot really tell if the file I uploaded was the correct one I wanted. I love the change in colors, but in the previous iteration, we got a little preview of the image rather than just the colors changing to not only confirm that it was uploaded, but that the correct one was uploaded. Let me know your thoughts!

Reply:
Great point, I've added a little preview to the currently displayed image:
![image](https://github.com/uchicago-cs/chigame/assets/55334507/36f92ef5-8c97-4354-a346-f1fb0e76825f)

and here's how it displays on the main page:
![image](https://github.com/uchicago-cs/chigame/assets/55334507/f69558d2-c06f-418c-b2a2-db68374cf4b3)